src/core/server/config/raw_config.ts

```diff
@@ -49,4 +49,11 @@ export interface RawConfig {
    * @returns List of the string config paths.
    */
   getFlattenedPaths(): string[];
+
+  /**
+   * Returns a full copy of the underlying raw config object. Should be used ONLY
+   * in extreme cases when there is no other better way, e.g. bridging with the
+   * "legacy" systems that consume and process config in a different way.
+   */
+  toRaw(): Record<string, any>;
```

Comment:
__note:__ will be need [here](https://github.com/elastic/kibana/pull/22190/files#diff-724505b9abf9bf469fed215250caf6f2R117), [here](https://github.com/elastic/kibana/pull/22190/files#diff-724505b9abf9bf469fed215250caf6f2R103) and [here](https://github.com/elastic/kibana/pull/22190/files#diff-724505b9abf9bf469fed215250caf6f2R59).